lib/MusicBrainz/Server/Controller/Root.pm

```diff
@@ -71,6 +71,9 @@ sub set_language : Path('set-language') Args(1)
     } else {
         # set the cookie to expire in a year
         $c->set_language_cookie($lang);
+        $c->flash->{message} =
```

Comment:
This shows a message just for this request, which can't be dismissed, but it disappears once you load another page.